internal/transport/http2_client.go

```diff
@@ -454,13 +454,21 @@ func newHTTP2Client(connectCtx, ctx context.Context, addr resolver.Address, opts
 	}
 
 	t.connectionID = atomic.AddUint64(&clientConnectionCounter, 1)
+	t.loopy = newLoopyWriter(clientSide, t.framer, t.controlBuf, t.bdpEst, t.conn, t.logger)
+	t.loopy.lWriterDone = grpcsync.NewEvent()
```

Comment:
2 comments, 

1. Why not init `lWriterDone` inside of `newLoopyWriter`
2. I think we can get away without the lWriterDone
  - Seems like this channel is required for blocking the client from calling conn.Close before we return from the loopywriter. We already have writeDone channel which is used for the same purpose.